nni/retiarii/integration.py

```diff
@@ -132,7 +132,8 @@ def handle_report_metric_data(self, data):
 
     @staticmethod
     def _process_value(value) -> Any:  # hopefully a float
-        value = json_tricks.loads(value)
+        if isinstance(value, str):
```

Comment:
here `value` must be `str`

Reply:
OK. In the test, it was a dict. I have converted it to str in the test. I also change L135 to "assert isintance(value, str)".